test/templates/helpers/cleanJsString.js

```diff
@@ -0,0 +1,29 @@
+var Lab = require('lab');
+var Code = require('code');
+
+var cleanJsString = require('../../../templates/helpers/cleanJsString');
+
+var lab = exports.lab = Lab.script();
+
+lab.experiment('Template Helper cleanJsString', function () {
+  lab.test('Test jsesc', function (done) {
+    Code.expect(cleanJsString('Ich ♥ Bücher'), 'Ich \\u2665 B\\xFCcher');
```

Comment:
Shouldn’t the tests expect the string wrapped in quotes now that `wrap: true` is used?